### DISCUSSION TITLE
Simple change to allow $this->replyToUser() to be used in commands called by callback queries.

### DISCUSSION THREAD

**Discussion by chuv1** [php-telegram-bot/core#1378] [Ideas]
Looks like a simple change from 
`$message = $this->getMessage() ?: $this->getEditedMessage()`
to
`$message = $this->getMessage() ?? $this->getEditedMessage() ?? $this->getCallbackQuery()`
on https://github.com/php-telegram-bot/core/blob/34b14599d28b71b24b400ef22659a7d1dc204a4d/src/Commands/Command.php#L441 will allow to use `$this->replyToUser()` in commands called by callback queries with `$this->getTelegram->executeCommand($command_name)`.
Right now it throws error, since $message->getFrom() is null in that case.

Pro? Cons?

**Comment by noplanman** [comment]
`$message = $this->getCallbackQuery()` would return a `CallbackQuery` entity instead of `Message`.

So theoretically, to get the `Message` object, it would have to be `$message = $this->getCallbackQuery()?->getMessage()`, but that may not be what is expected.

However, it should be safe to expect that the user calling the callback is who should be replied to, not the user who sent the initial message.

@TiiFuchs Any thoughts?
@chuv1 Feel free to PR this :ok_hand:
